wgpu-hal/src/metal/adapter.rs

```diff
@@ -622,54 +591,50 @@ impl super::PrivateCapabilities {
             sample_count_mask |= 8;
         }
 
+        let rw_texture_tier = if version.at_least(os_is_mac, (10, 13).into(), 11.into()) {
```

Comment:
looks like we could make `at_least` accept generic argument(s) converting into the version